.orchestra/config/lib/cmake.lib.yml

```diff
@@ -101,6 +100,8 @@
     fi
     (@ end -@)
 
+    cmake --install "$BUILD_DIR"
+
     (@- if post_install_script: @)
```

Comment:
I think `@` is eating spaces.

```
[+] INFO - The following script is going to be executed:
cmake --build "$BUILD_DIR" ${JOBS:+-j$JOBS}cmake --install "$BUILD_DIR"
```

Which leads to:

```
[+] ERROR - Script failed with exit code 1
Output stream:
'-j' invalid number '8cmake' given.
Unknown argument --install
Unknown argument /builds/gitlab/revng/orchestra/orchestra/build/libfort/optimized
```

Test things!

Reply:
It's not the `@`, but the `-`, and yep, this one should be removed `(@- if post_install_script: @)`

Reply:
Actually, also the one in `(@ end -@)`

Reply:
Can you push a fix?